src/main/webapp/loginScript.js

```diff
@@ -1,10 +1,20 @@
 function onSignIn(googleUser) {
   var profile = googleUser.getBasicProfile();
-  localStorage.setItem("username",profile.getName()); //sends player's names into local storage
-  localStorage.setItem("ID", profile.getId()); //sends player's id into local storage
+  var name = profile.getName();
+  var id = profile.getId();
```

Comment:
We're using this ID as a key in Datastore. But are you sure you get the same ID every time you log in with your Google account? If it is the same every time, then it's not safe to use it as a login cookie the way we have been doing.